config/cert-manager/kustomization.yaml

```diff
@@ -6,19 +6,13 @@ resources:
 helmCharts:
   - name: cert-manager
     repo: https://charts.jetstack.io
-    version: v1.7.1
+    version: v1.12.2
     releaseName: mgc
     namespace: cert-manager
     valuesInline:
       global:
         leaderElection:
           namespace: cert-manager
       installCRDs: true
-      # The Job API is not present
-      startupapicheck:
```

Comment:
Not having this enabled leaves out a valid check that may not manifest until later?